ts/src/bingx.ts

```diff
@@ -100,11 +100,13 @@ export default class bingx extends Exchange {
                                 'trade/openOrders': 3,
                                 'trade/historyOrders': 3,
                                 'account/balance': 3,
+                                'ticker/24hr': 1,
```

Comment:
@sc0Vu maybe we can implement some methods with these endpoints? (fetchTicker spot and cancelOrders spot). Will check it